change/@ni-nimble-components-ba01bf12-98d0-4b54-8563-7bea754d9578.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Added @fredvisser and @mollykreis as reviewers.

@mollykreis in particular the [follow-up from the last PR](https://github.com/ni/nimble/pull/2065#discussion_r1587890332).
@fredvisser in particular for the direction I'm taking the README, i.e. de-emphasizing the tokens package and language around the listing of the spright packages.